pkgs/by-name/ac/act/package.nix

```diff
@@ -1,20 +1,21 @@
-{ lib
-, fetchFromGitHub
-, buildGoModule
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
 }:
 
 buildGoModule rec {
```

Comment:
Would you like to define `version` to remove this `rec` keyword? (ref: https://nix.dev/guides/best-practices#recursive-attribute-set-rec)

Reply:
Done 👍